src/modules/studyMembers/studyMember.module.ts

```
@@ -0,0 +1,14 @@
+import { Module, forwardRef } from '@nestjs/common';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)